sdk/docs/manually-written/overview/explanations/ledger-model/interoperability.rst

```diff
@@ -21,13 +21,13 @@ Synchronizer-aware projection
 
 Certain Daml ledgers can interoperate with other Daml ledgers.
 That is, the contracts created on one ledger can be used and archived in transactions on other ledgers.
-Some Participant Nodes can connect to multiple ledgers and provide their parties unified access to those ledgers via the :externalref:`Ledger API <ledger-api-services>`.
+Some Participant Nodes can connect to multiple ledgers and provide their parties unified access to those ledgers via the :externalref:`gRPC Ledger API <ledger-api-services>`.
```

Comment:
I would expect that we link to the "Ledger API overview" that can explain that there is one Ledger API accessible via two protocols. Calling out gRPC goes in the wrong direction here.

Reply:
Let's at least a TODO at the top reminding us that this content needs to be rewritten to only talk about the Canton ledger.

Reply:
I have gone way to far with this PR already. It was supposed to be about the renames, but I couldn't resist the temptation to set things right. I am happy to make those changes in the follow-up that I already started on here:
https://github.com/digital-asset/daml/pull/21266